usaspending_api/recipient/migrations/0006_auto_20210315_2028.py

```diff
@@ -0,0 +1,51 @@
+# Generated by Django 2.2.18 on 2021-03-15 20:28
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('recipient', '0005_auto_20210216_2120'),
+    ]
+
+    operations = [
+        migrations.AddIndex(
+            model_name='recipientprofile',
+            index=models.Index(fields=['last_12_months'], name='recipient_p_last_12_535d0c_idx'),
+        ),
+        migrations.AddIndex(
+            model_name='recipientprofile',
+            index=models.Index(fields=['last_12_contracts'], name='recipient_p_last_12_1b3d9f_idx'),
+        ),
+        migrations.AddIndex(
+            model_name='recipientprofile',
+            index=models.Index(fields=['last_12_grants'], name='recipient_p_last_12_3776b8_idx'),
+        ),
+        migrations.AddIndex(
+            model_name='recipientprofile',
+            index=models.Index(fields=['last_12_direct_payments'], name='recipient_p_last_12_eef9af_idx'),
+        ),
+        migrations.AddIndex(
+            model_name='recipientprofile',
+            index=models.Index(fields=['last_12_loans'], name='recipient_p_last_12_11e0b3_idx'),
+        ),
+        migrations.AddIndex(
+            model_name='recipientprofile',
+            index=models.Index(fields=['last_12_other'], name='recipient_p_last_12_7a7440_idx'),
+        ),
+        # Performed with RAW SQL because Django migrations don't support 'NULLS LAST'
+        migrations.RunSQL(
+            "CREATE INDEX recipient_profile_recipient_name_d ON public.recipient_profile USING btree (recipient_name DESC NULLS LAST)",
+            reverse_sql="DROP INDEX recipient_profile_recipient_name_d"
+        ),
+        migrations.RunSQL(
+            "CREATE INDEX recipient_profile_recipient_unique_id_d ON public.recipient_profile USING btree (recipient_unique_id DESC NULLS LAST)",
+            reverse_sql="DROP INDEX recipient_profile_recipient_unique_id_d"
+        ),
+        # Performed with RAW SQL because Django migrations don't support GinIndexes with a specific Postgres operator class:
```

Comment:
Q: So this then probably means there was another earlier custom RunSQL migration that applied GinIndex to the `recipient_name` column?

Reply:
Yes, there is a comment in the model calling out specific migrations with manual SQL being applied.